python/triton/testing.py

```diff
@@ -17,16 +19,38 @@ def nvsmi(attrs):
     return ret
 
 
+def _quantile(a, q):
```

Comment:
Need a comment to note why we need a custom implementation instead of using torch/numpy

Reply:
added